pkg/common/cns-lib/vsphere/vslm.go

```diff
@@ -0,0 +1,63 @@
+/*
+Copyright 2021 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package vsphere
+
+import (
+	"context"
+
+	"github.com/vmware/govmomi/vim25"
+	"github.com/vmware/govmomi/vslm"
+	"sigs.k8s.io/vsphere-csi-driver/pkg/csi/service/logger"
+)
+
+// NewVslmClient creates a new Vslm client
+func NewVslmClient(ctx context.Context, c *vim25.Client) (*vslm.Client, error) {
+	log := logger.GetLogger(ctx)
+	vslmClient, err := vslm.NewClient(ctx, c)
+	if err != nil {
+		log.Errorf("failed to create a new client for Vslm. err: %v", err)
+		return nil, err
+	}
+	return vslmClient, nil
+}
+
+// ConnectVslm creates a Vslm client for the virtual center.
+func (vc *VirtualCenter) ConnectVslm(ctx context.Context) error {
```

Comment:
didn't govmomi have this? 
may be this should go into govmomi repo.

Reply:
This we will use in our repo similar to: https://github.com/kubernetes-sigs/vsphere-csi-driver/blob/44b5dff2604f965ae969a61e77e4fd5c4582b203/pkg/common/cns-lib/vsphere/cns.go#L39